volume17.xml

```diff
@@ -0,0 +1,215 @@
+                </dateline>
+                <p>52 miles. Arrived at <placeName ref="Assiut">Assiut</placeName> early, and waited for various things &#8211; 4 hours &#8211; then on to 7 miles of <placeName>Hadj Kandul.</placeName></p>
+                <dateline>
+                    <date when="1911-03-09">Thursday. March 9th.</date>
+                </dateline>
+                <p>Started at 7 o'clock &#8211; stopped at <persName>Hagi Kondul</persName> for an hour <pb n="82"/>so that <persName>El&#232;na</persName> should see the <orgName>Khuenaton</orgName> pavement, then on, until <persName>Beni Hassan</persName> was reached, which <persName>El&#232;na</persName> and <persName>Harry</persName> visited. The day hot &#8211; wind and clouds of dust. 43 miles.</p>
+                <dateline>
+                    <date when="1911-03-09">Friday, March 10th.</date>
+                </dateline>
+                <p>53 miles. Started at 7 o'clock &#8211; spent 2 hours at <placeName>Minia</placeName> &#8211; tied up at 6 o'clock at <placeName>Maghaga.</placeName></p>
+                <dateline>
+                    <date when="1911-03-11">Saturday. Mar. 11th.</date>
+                </dateline>
+                <p>87 miles. An early start at 6 o'clock &#8211; and reached <name>Bedreschein</name> at 7 o'clock. The river has been rather good. The tug has been on the sand once or twice for a short time, but the dahabya has not once touched it &#8211; a strong wind all day and cold.</p>
+                <dateline>
+                    <date when="1911-03-12">Second Sunday in Lent. March 12th.</date>
+                </dateline>
+                <p>Started at 7 o'clock &#8211; and made the bridge at 9 o'clock and waited an hour to pass. Came to our landing at 11.30 &#8211; and found the mooring garden looking very pretty. We arrived the 10th day from <placeName>Luxor.</placeName> Found telegram telling of <persName>Harold Jones'</persName> peaceful end on Thursday, the 9th.</p>
+                <dateline>
+                    <date when="1911-03-12">March 13th. Monday.</date>
+                </dateline>
+                <p>Mr. <persName>Draper</persName> to lunch yesterday. At home all day &#8211; rather warm today. <persName>El&#232;na</persName> and I drove into town this morning &#8211; saw a number <pb n="83"/>of friends &#8211; met <persName>Mr. Newberry</persName> at <placeName>Deimers,</placeName> and he went with us to <persName>Kolekians</persName> and to <persName>Kasseras</persName> &#8211; there <persName ref="#Davis_Theodore_M">Theo</persName> and <persName>Harry</persName> joined us, and we went to Nachmans. <persName>El&#232;na</persName> and <persName>Harry</persName> went after lunch to the <placeName>Mena House,</placeName> and <persName ref="#Davis_Theodore_M">Theo</persName> and I went for a drive &#8211; and finding ourselves, in spite of warm wraps, freezing in the cold wind, we came home.</p>
```

Comment:
several places in need of place tag (some tagged as pers here) that are not in indices. Nachmans in need of tag.